app/workers/structured_data/process_data_job.rb

```diff
@@ -17,18 +16,31 @@ def perform(saved_claim_id)
         @claim = SavedClaim.find(saved_claim_id)
 
         relationship_type = @claim.parsed_form['relationship']&.fetch('type', nil)
+        claimant_name = @claim.parsed_form['claimantFullName']
+        claimant_address = @claim.parsed_form['claimantAddress']
+
         veteran = BipClaims::Service.new.lookup_veteran_from_mvi(@claim)
+
+        claimant = lookup_claimant(relationship_type, veteran, claimant_name, claimant_address) if veteran
       ensure
         @claim.process_attachments! # upload claim and attachments to Central Mail
 
         # veteran lookup for hit/miss metrics in support of Automation work
         StatsD.increment("#{stats_key}.success", tags: [
                            "relationship:#{relationship_type}",
-                           "veteranInMVI:#{veteran&.participant_id}"
+                           "veteranInMVI:#{veteran&.participant_id.present?}",
+                           "claimantFound:#{claimant.present?}"
```

Comment:
If the code in the `begin` block raises somehow, then `relationship_type`, `veteran`, & `claimant`, could all be `nil`. I think we want the code in the `ensure` block to not be dependent on anything going on inside the `begin` block.

Reply:
I'm using `present?` here so that our stats trigger with a false/not found if those values are `nil`—does that work?